tests/accessor/accessor_common.h

```diff
@@ -365,22 +500,47 @@ void check_zero_dim_constructor(GetAccFunctorT get_accessor_functor) {
           auto acc = get_accessor_functor(data_buf, cgh);
           if constexpr (Target == sycl::target::host_task) {
             cgh.host_task([=] {
-              read_write_zero_dim_acc<DataT, AccessMode>(acc, res_acc);
+              // We are free either to create new accessor instance or to
+              // modify original accessor and provide reference to it;
+              // a reference to original accessor would be used if there was
+              // no any modify_accessor functor passed
+              auto&& acc_instance =
+                  ((detail::invoke_helper{modify_accessor}) = ... = acc);
+              read_write_zero_dim_acc<DataT, AccessMode>(acc_instance, res_acc);
             });
           } else if constexpr (Target == sycl::target::device) {
             cgh.parallel_for_work_group(r, [=](sycl::group<1>) {
-              read_write_zero_dim_acc<DataT, AccessMode>(acc, res_acc);
+              auto&& acc_instance =
+                  ((detail::invoke_helper{modify_accessor}) = ... = acc);
```

Comment:
Are these `()` around `detail::invoke_helper{modify_accessor}` required for `=` folding?

Reply:
No, `()` are don't required around `detail::invoke_helper{modify_accessor}`. removed [here](https://github.com/KhronosGroup/SYCL-CTS/pull/339/commits/558b066913e80167683ea3c121e2324dd68a0c8a#diff-8067235ffc00be4a68e8da8448acbf789b606099c5dbc1e969df066646bd2f64R508)